app/views/v1/inventories/show.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.partial! "inventory", inventory: @inventory
```

Comment:
Add new line at the end of this file:
http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline
http://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file